internal/compiler/widgets/common/menu-base.slint

```diff
@@ -168,5 +170,8 @@ export component MenuItemBase {
             background-layer.background: root.current-background;
             label.color: root.current-foreground;
         }
+        disabled when !entry.enabled : {
+            label.opacity: 0.5;
```

Comment:
Not sure of this is really how it should look on all style, but that should do it for now.